projects/project1/Descriptive Statistics Draft.ipynb

```diff
@@ -0,0 +1,440 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Draft Review"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This is some initial review of the dataset using some basic descriptive statistics techniques in python."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import standard libraries\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "from sklearn.datasets import fetch_california_housing\n",
+    "from scipy.stats import dgamma\n",
+    "import matplotlib.pyplot as plt"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "  InvoiceNo StockCode                          Description  Quantity  \\\n",
+      "0    536365    85123A   WHITE HANGING HEART T-LIGHT HOLDER         6   \n",
+      "1    536365     71053                  WHITE METAL LANTERN         6   \n",
+      "2    536365    84406B       CREAM CUPID HEARTS COAT HANGER         8   \n",
+      "3    536365    84029G  KNITTED UNION FLAG HOT WATER BOTTLE         6   \n",
+      "4    536365    84029E       RED WOOLLY HOTTIE WHITE HEART.         6   \n",
+      "\n",
+      "          InvoiceDate  UnitPrice  CustomerID         Country  \n",
+      "0 2010-12-01 08:26:00       2.55     17850.0  United Kingdom  \n",
+      "1 2010-12-01 08:26:00       3.39     17850.0  United Kingdom  \n",
+      "2 2010-12-01 08:26:00       2.75     17850.0  United Kingdom  \n",
+      "3 2010-12-01 08:26:00       3.39     17850.0  United Kingdom  \n",
+      "4 2010-12-01 08:26:00       3.39     17850.0  United Kingdom  \n"
+     ]
+    }
+   ],
+   "source": [
+    "# Read the Excel file\n",
+    "df = pd.read_excel('datasets/online+retail/Online Retail.xlsx', engine='openpyxl')\n",
```

Comment:
is this assuming you have the dataset locally?

Reply:
This is assuming the dataset is in the repository files of the project repository. Happy to do it another way if you'd prefer.